TFC Integrations - ServiceNow Service Catalog - version update and Agents Support feature (#322)
hashicorp/terraform-docs-common: website/docs/cloud-docs/integrations/service-now/service-catalog.mdx

[VCS-driven run workflow](/terraform/cloud-docs/run/ui). It is important to keep in mind
that all of the ServiceNow workspaces connected to a given repository will be
updated whenever changes are merged to the associated branch in that repository.

## Execution Mode

Starting from version 2.2.0, the Service Catalog app allows you to set the [execution mode](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings#execution-mode) for your Terraform Workspaces.

 - The default value is "Remote", which performs runs on Terraform Cloud's infrastructure. 
 - Changing the execution mode to "Local" allows you to execute Terraform runs locally using the CLI workflow. 

>  - Selecting "Agent" enables you to run Terraform operations on isolated, private, or on-premises infrastructure. This option requires an Agent Pool to be created in your Terraform Organization beforehand. Provide its ID in the "Agent Pool ID" field when you order a new Workspace through the Service Catalog.

Reviewer: Styling suggestion for clarity:
```suggestion
 - Selecting "Agent" mode allows you to run Terraform operations on isolated, private, or on-premises infrastructure. This option requires you to create an Agent Pool in your organization beforehand, then provide that Agent Pool's id when you order a new workspace through the Service Catalog.
```
